apps/src/templates/MarketingSkinnyBanner.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import DCDO from '@cdo/apps/dcdo';
+
+// Not using inline styles to keep regular and responsive styles together.
+//   Copied styles from 'shared/css/csa_skinny_banner.scss' that's used
+//   in the Pegasus version of this banner:
+//  'pegasus/sites.v3/code.org/views/csa_skinny_banner.haml'.
+import './marketingSkinnyBanner.scss';
+
+const IMAGE_BASE_URL = '/blockly/media/';
+
+// Skinny banner component
+// Current: CSA Launch May 2022
+export default function MarketingSkinnyBanner() {
+  const header_text = 'A new approach to AP® CSA!';
+  const text =
+    'Creative projects and real world connections in our equity-driven curriculum';
+
+  // Only show if DCDO flag is set to true,
```

Comment:
Nit: no need to leave this comment, it's clear from the code :)

Reply:
I'll remove it!